<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTaskClass</class>
 <widget class="QWidget" name="AddTaskClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddTask</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QWidget" name="wgt_top" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>48</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(251, 251, 251);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_15" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/FaultInjection/images/icon1.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lb_title_2">
           <property name="minimumSize">
            <size>
             <width>291</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>291</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>新建任务岗位配置</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>845</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_close_2">
           <property name="minimumSize">
            <size>
             <width>31</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>31</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:hover{
	background-color:transparent;
	color: rgb(255, 85, 0);
	background-image: url(:/FaultInjection/images/关闭-悬浮.png);
    border:0px;}
QPushButton:pressed{
	background-color:transparent; 
	background-image: url(:/FaultInjection/images/关闭-点击.png);
    border:0px;}
QPushButton {
	background-color:transparent;
	background-image: url(:/FaultInjection/images/关闭-默认.png);
    border:0px;
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_16" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>2</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>2</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(235, 235, 235);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgt_center" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLineEdit" name="le_roketName">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>476</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:5px;padding:2px 4px;
font: 12pt &quot;微软雅黑&quot;;
</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>202</x>
        <y>40</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>火箭型号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_taskName">
      <property name="geometry">
       <rect>
        <x>198</x>
        <y>180</y>
        <width>476</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:5px;padding:2px 4px;
font: 12pt &quot;微软雅黑&quot;;
</string>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>任务岗位</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_roketSoftName">
      <property name="geometry">
       <rect>
        <x>198</x>
        <y>290</y>
        <width>476</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:5px;padding:2px 4px;
font: 12pt &quot;微软雅黑&quot;;
</string>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>250</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>岗位软件名称</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgt_bottom" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>48</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_17" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>2</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>2</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(235, 235, 235);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_18" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1059</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_cancel_2">
           <property name="minimumSize">
            <size>
             <width>61</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>61</width>
             <height>31</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:hover{
 	background-color:transparent;
	border-image: url(:/FaultInjection/images/bt_suspension.png);
	color: rgb(255, 255, 255);
	font: 12pt &quot;微软雅黑&quot;;
    border:0px;}
QPushButton:pressed{
    background-color:transparent; 
	border-image: url(:/FaultInjection/images/bt_disable.png);
	color: rgb(255, 255, 255);
	font: 12pt &quot;微软雅黑&quot;;
    border:0px;}
QPushButton {
  	background-color:transparent;
	border-image: url(:/FaultInjection/images/bt_normal.png);
	color: rgb(255, 255, 255);
	font: 12pt &quot;微软雅黑&quot;;
    border:0px;
}</string>
           </property>
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>17</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_ok_2">
           <property name="minimumSize">
            <size>
             <width>61</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>61</width>
             <height>31</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:hover{
 	background-color:transparent;
	border-image: url(:/FaultInjection/images/bt_suspension.png);
	color: rgb(255, 255, 255);
	font: 12pt &quot;微软雅黑&quot;;
    border:0px;}
QPushButton:pressed{
    background-color:transparent; 
	border-image: url(:/FaultInjection/images/bt_disable.png);
	color: rgb(255, 255, 255);
	font: 12pt &quot;微软雅黑&quot;;
    border:0px;}
QPushButton {
  	background-color:transparent;
	border-image: url(:/FaultInjection/images/bt_normal.png);
	color: rgb(255, 255, 255);
	font: 12pt &quot;微软雅黑&quot;;
    border:0px;
}</string>
           </property>
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FaultInjection.qrc"/>
 </resources>
 <connections/>
</ui>
